README.md

```diff
@@ -21,30 +21,13 @@ This is just one tool in the chain. `notes` is a command line tool, and some peo
 
 ## How do I install this?
 
-Download `notes`, `chmod +x`, put it in your `$path`. This will probably do it:
```

Comment:
This goes double for the bash completion section, which is still very relevant to anybody on OSX or probably Windows, as well as anybody with other non-standard setups.